src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -426,6 +428,7 @@ export class Evaluator {
       this.nextPositionsToUpdate.addMany(this.getCellsDependingOn([child]));
       this.nextPositionsToUpdate.addMany(this.getArrayFormulasBlockedBy(child));
     }
+    this.nextPositionsToUpdate.delete(position);
```

Comment:
Still needed even after https://github.com/odoo/o-spreadsheet/commit/face8ee024d17ff9f0124b8194f0fd3b3faeb420 ?
What added `position` in the set in the first place ?